### PR TITLE
webview: api to get host browser window

### LINKED ISSUE #4481: Add a way to get host BrowserWindow of webview
Ive worked around this but spent some time puzzling.. my webview sends an ipc message to the main process.. now i want to do something to the window containing the webview.. if i use fromWebContents I get undefined.

A similar issue occurs if i want to broadcast an ipc message to all webcontents and webviews, I must enumerate browserWindows, then write code in each browserWindow that receives the message and forwards to every webview.

### PR DESCRIPTION
**Pull request by deepak1556**
Fixes #4481

### CODE REVIEW

**Comment by zcbenz** [comment]
We can provide a new API to get the WebContents of webview, and then a new API to get the host WebContents from it. Users can then use `BrowserWindow.fromWebContents` to get the window.

This is more complicated than a single `getOwnerBrowserWindow` API, but more general.

**Comment by deepak1556** [comment]
Have added a method on webview to retrieve its webcontents and added a property to `WebContents` to retrieve its embedder if available.

File: atom/browser/api/atom_api_web_contents.h [hunk 1]
```diff
@@ -282,11 +283,15 @@ class WebContents : public mate::TrackableObject<WebContents>,
                              IPC::Message* message);
 
   v8::Global<v8::Value> session_;
+  v8::Global<v8::Value> embedder_web_contents_;
```
Comment (zcbenz) [inline atom/browser/api/atom_api_web_contents.h]: This is not used.

File: docs/api/web-contents.md [hunk 2]
```diff
@@ -842,6 +842,10 @@ win.webContents.on('did-finish-load', function() {
 
 Returns the [session](session.md) object used by this webContents.
 
+### `webContents.embedderWebContents`
```
Comment (zcbenz) [inline docs/api/web-contents.md]: Probably use `hostWebContents` instead, which matches the `sendToHost` API.

Comment (deepak1556) [inline docs/api/web-contents.md]: Fixed, thanks!

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**